apps/meteor/ee/app/license/server/getStatistics.ts

```diff
@@ -31,6 +31,25 @@ export async function getStatistics(): Promise<ENTERPRISE_STATISTICS> {
 			statistics.seatRequests = count;
 		}),
 	);
+
+	if (hasLicense('livechat-enterprise')) {
+		const eeModelsStats = await getStatsFromEEModels();
+		statistics = {
+			...statistics,
+			...eeModelsStats,
+		};
+	}
```

Comment:
Move this check to inside of "getStatsFromEEModels" func

Reply:
Done :heavy_check_mark: